docs/source/markdown/podman-build.1.md

```diff
@@ -650,6 +650,10 @@ If --userns-gid-map-group is specified, but --userns-uid-map-user is not
 specified, `podman` will assume that the specified group name is also a
 suitable user name to use as the default setting for this option.
 
+**NOTE:** When this option is specified by a rootless user, the specified
+mappings are relative to the rootless usernamespace in the container, rather
+than being relative to the host as it would be when run rootlful.
+
```

Comment:
typo in rootlful

Reply:
Drats,
I cut/paste that directly from Buildah.  Need to fix it there.  TY and nice Eagle eye.

Reply:
fixed